bin/list-all

```diff
@@ -2,7 +2,7 @@
 
 function sort_versions() {
   sed 'h; s/[+-]/./g; s/.p\([[:digit:]]\)/.z\1/; s/$/.z/; G; s/\n/ /' | \
-    LC_ALL=C sort -gt. -k 1,1 -k 2,2n -k 3,3n -k 4,4n -k 5,5n | sort -r - | awk '{print $2}'
+    LC_ALL=C sort -gt. -k 1,1 -k 2,2n -k 3,3n -k 4,4n -k 5,5n | sort -rV - | awk '{print $2}'
```

Comment:
```suggestion
    LC_ALL=C sort -gt. -k 1,1 -k 2,2n -k 3,3n -k 4,4n -k 5,5n | awk '{arr[NR]=$0} END {for (i=NR; i>0; i--) {split(arr[i], a, " "); print a[2]}}'
```

Adapted from:
https://stackoverflow.com/a/48362923

Reply:
Thanks for looking into this! I didn't have a chance to sit down until now, but glad it got sorted out (no pun intended)!